### PR TITLE
updates per https://phabricator.wikimedia.org/T246912

### PR DESCRIPTION
**Pull request by geohci**
* Cleaned up History and Society
* Removed a number of WikiProjects from Culture.Biography (Pashtun, Taiwan, Assyria, Azerbaijan, Basque, Berbers, Igbo) as they were determined to not be predominantly biographies and already be grouped in their respective Geography topics.
* Removed duplicate WikiProjects in tree (Women's sport)
* Recompiled datasets based on new taxonomy
* Removed taxonomy.yaml as that was a duplicate of datasets/wikiproject_taxonomy.halfak_20191202.yaml but not part of the Makefile so liable to become out of date

See https://phabricator.wikimedia.org/T246912#5982179 for details about specific WikiProjects and https://phabricator.wikimedia.org/T246912#6006350 for a link to a spreadsheet that shows how a given WikiProject's articles are spread across topics.

### CODE REVIEW

File: datasets/wikiproject_taxonomy.halfak_20191202.yaml [hunk 1]
```diff
@@ -27,13 +27,7 @@ Culture:
      - WikiProject African diaspora
      - WikiProject Asian Americans
      - WikiProject Assyria
-     - WikiProject Azerbaijan
-     - WikiProject Basque
-     - WikiProject Berbers
-     - WikiProject Igbo
      - WikiProject Franco-Americans
-     - WikiProject Pashtun
-     - WikiProject Taiwan
```
Comment (halfak) [inline datasets/wikiproject_taxonomy.halfak_20191202.yaml:36]: Why are these removed?  No hits?

File: datasets/wikiproject_taxonomy.halfak_20191202.yaml [hunk 2]
```diff
@@ -43,7 +37,6 @@ Culture:
      - WikiProject Jewish Women
      - WikiProject Women in Music
      - WikiProject Women in Technology
-     - WikiProject Women's sport
```
Comment (halfak) [inline datasets/wikiproject_taxonomy.halfak_20191202.yaml:46]: Same here.  Why remove this?

Comment (halfak) [reply datasets/wikiproject_taxonomy.halfak_20191202.yaml:46]: Aha!  Duplicate?

File: datasets/wikiproject_taxonomy.halfak_20191202.yaml (hunk 1 above)
Comment (geohci) [reply datasets/wikiproject_taxonomy.halfak_20191202.yaml:36]: For all of these that I suggested to remove, there seems to be a large number of articles that aren't biographies, which makes me think they probably add more noise than signal and are best left in their respective geography categories. I've been using this spreadsheet (https://docs.google.com/spreadsheets/d/1uXelaSPyR6EGmQoUKkf5rsXuMXBNxxqyHKFPdFEn_tc/edit?usp=sharing) as a guide for what is covered by a given WikiProject (it's a distribution of which topics would be applied to articles in a WikiProject if it was removed from the taxonomy so it captures what other sorts of WikiProjects tag the same articles) as well as the What Links Here for the template. For Igbo, Pashtun, Taiwan, etc. the majority don't seem to be biographies but articles about Igbo/Pashtun/Taiwanese places, culture, and history. Not a hard rule obviously, but felt like they were probably adding more noise than signal to the Biography category and if these were going to be included in Biography, we probably should include a bunch of other ethnicity-oriented wikiprojects. WikiProject Berbers probably has the most biographical focus but even that has a lot of non-biography articles (https://en.wikipedia.org/wiki/Special:WhatLinksHere/Template:WikiProject_Berbers). And actually, I think under the same criteria, I probably would remove WikiProject Assyria as well. Thoughts?

File: datasets/wikiproject_taxonomy.halfak_20191202.yaml (hunk 2 above)
Comment (geohci) [reply datasets/wikiproject_taxonomy.halfak_20191202.yaml:46]: yeah -- just happened to notice the duplicate while looking at other things so removed it

File: datasets/wikiproject_taxonomy.halfak_20191202.yaml (hunk 1 above)
Comment (halfak) [reply datasets/wikiproject_taxonomy.halfak_20191202.yaml:36]: That makes sense.  We should document this in the commit comments or somewhere else.  I'm open to ideas for how we can help future people understand the change.

**Comment by halfak** [comment]
This looks good to me.  If you want to merge now, consider this my +1.  I saw that @geohci was considering cutting WikiProject Assyria too and I also agree with that but will wait on that to merge myself.

File: datasets/wikiproject_taxonomy.halfak_20191202.yaml (hunk 1 above)
Comment (geohci) [reply datasets/wikiproject_taxonomy.halfak_20191202.yaml:36]: good point -- I'll update the commit notes as that seems the most straightforward way. I guess eventually, a release notes with actual versioning of the taxonomy makes the most sense to me?

**Comment by geohci** [comment]
Sigh...and I now realize why it seems you were suggesting caution around making the Assyria change. Made that change but only did it in one file (the final compiled taxonomy) and not the source taxons (culture.yaml). I'll go back and create another pull request to fix that unless there's a better/easier way :/
